core/src/test/java/org/opensearch/sql/expression/datetime/DateTimeFunctionTest.java

```diff
@@ -479,6 +480,76 @@ public void dayOfYear() {
     assertEquals(integerValue(220), eval(expression));
   }
 
+  public void test_day_of_year(String date, int dayOfYear) {
+    FunctionExpression expression = DSL.day_of_year(DSL.literal(new ExprDateValue(date)));
+    assertEquals(INTEGER, expression.type());
+    assertEquals(integerValue(dayOfYear), eval(expression));
+  }
+
+  @Test
+  public void day_Of_Year() {
+    lenient().when(nullRef.valueOf(env)).thenReturn(nullValue());
+    lenient().when(missingRef.valueOf(env)).thenReturn(missingValue());
+
+    FunctionExpression expression = DSL.day_of_year(DSL.literal(new ExprDateValue("2020-08-07")));
+    assertEquals(INTEGER, expression.type());
+    assertEquals("day_of_year(DATE '2020-08-07')", expression.toString());
+    assertEquals(integerValue(220), eval(expression));
+
+    expression = DSL.day_of_year(DSL.literal("2020-08-07"));
+    assertEquals(INTEGER, expression.type());
+    assertEquals("day_of_year(\"2020-08-07\")", expression.toString());
+    assertEquals(integerValue(220), eval(expression));
+
+    expression = DSL.day_of_year(DSL.literal("2020-08-07 01:02:03"));
+    assertEquals(INTEGER, expression.type());
+    assertEquals("day_of_year(\"2020-08-07 01:02:03\")", expression.toString());
+    assertEquals(integerValue(220), eval(expression));
+
+    //28th of Feb
+    test_day_of_year("2020-02-28", 59);
+
+    //29th of Feb during leap year
+    test_day_of_year("2020-02-29 23:59:59", 60);
+    test_day_of_year("2020-02-29", 60);
```

Comment:
Try using `assertAll` or split into different test cases

Reply:
Fixed in https://github.com/Bit-Quill/opensearch-project-sql/pull/173/commits/9c3c1f18a471900bea402048557ac187443e4763